vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -1518,4 +1522,108 @@ winrt::Microsoft::ReactNative::ComponentView WindowsTextInputComponentView::Crea
   return winrt::make<WindowsTextInputComponentView>(compContext, tag, reactContext);
 }
 
+void WindowsTextInputComponentView::autoCapitalizeOnUpdateProps(const std::string &capitalizationType) noexcept {
+  /*
+    Possible values are:
+     Characters - All characters.
+     Words - First letter of each word.
+     Sentences - First letter of each sentence.
+     None - Do not autocapitalize anything.
+  */
+
+  BSTR bstr = nullptr;
```

Comment:
Yes, wil::unique_bstr. Initially I wanted to install WIL nupkg but it fails due to a conflict with another package, so to avoid delaying the work I opted to do manual allocation, as other parts of the code already do.